[2023-09-10T17:26:17.195+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: website_crawler.create_data_table manual__2023-09-10T17:25:58.704390+00:00 [queued]>
[2023-09-10T17:26:17.239+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: website_crawler.create_data_table manual__2023-09-10T17:25:58.704390+00:00 [queued]>
[2023-09-10T17:26:17.244+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-09-10T17:26:17.298+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_data_table> on 2023-09-10 17:25:58.704390+00:00
[2023-09-10T17:26:17.340+0000] {standard_task_runner.py:57} INFO - Started process 66 to run task
[2023-09-10T17:26:17.354+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'website_crawler', 'create_data_table', 'manual__2023-09-10T17:25:58.704390+00:00', '--job-id', '508', '--raw', '--subdir', 'DAGS_FOLDER/crawl_data_log_into_postgres.py', '--cfg-path', '/tmp/tmp7_00puzm']
[2023-09-10T17:26:17.368+0000] {standard_task_runner.py:85} INFO - Job 508: Subtask create_data_table
[2023-09-10T17:26:17.543+0000] {task_command.py:415} INFO - Running <TaskInstance: website_crawler.create_data_table manual__2023-09-10T17:25:58.704390+00:00 [running]> on host 30cb2bd58730
[2023-09-10T17:26:17.754+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='website_crawler' AIRFLOW_CTX_TASK_ID='create_data_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-10T17:25:58.704390+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-10T17:25:58.704390+00:00'
[2023-09-10T17:26:17.761+0000] {sql.py:274} INFO - Executing: -- create data table
CREATE TABLE IF NOT EXISTS test2(
    title CHAR(5000) NOT NULL,
    author CHAR(5000) NOT NULL,
    date CHAR(5000) NOT NULL,
    time CHAR(5000) NOT NULL,
    summary CHAR(5000) PRIMARY KEY);
[2023-09-10T17:26:17.790+0000] {base.py:73} INFO - Using connection ID 'postgres_local' for task execution.
[2023-09-10T17:26:17.817+0000] {base.py:73} INFO - Using connection ID 'postgres_local' for task execution.
[2023-09-10T17:26:17.835+0000] {sql.py:418} INFO - Running statement: -- create data table
CREATE TABLE IF NOT EXISTS test2(
    title CHAR(5000) NOT NULL,
    author CHAR(5000) NOT NULL,
    date CHAR(5000) NOT NULL,
    time CHAR(5000) NOT NULL,
    summary CHAR(5000) PRIMARY KEY);, parameters: None
[2023-09-10T17:26:17.899+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=website_crawler, task_id=create_data_table, execution_date=20230910T172558, start_date=20230910T172617, end_date=20230910T172617
[2023-09-10T17:26:17.973+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-10T17:26:18.028+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
